app/experimenter/experiments/tests/api/v6/test_serializers.py

```diff
@@ -54,9 +54,9 @@ def test_expected_schema_with_desktop_single_feature(self):
                 # DRF manually replaces the isoformat suffix so we have to do the same
                 "startDate": experiment.start_date.isoformat().replace("+00:00", "Z"),
                 "targeting": (
-                    'browserSettings.update.channel == "nightly" '
-                    "&& version|versionCompare('94.!') >= 0 "
-                    "&& 'app.shield.optoutstudies.enabled'|preferenceValue"
```

Comment:
Should we be testing both cases (with and without parens)?

Reply:
No no there won't be a non paren case anymore